src/XRP/model/xrp-path-element.ts

```diff
@@ -19,8 +20,28 @@ export default class XrpPathElement {
   public static from(pathElement: PathElement): XrpPathElement {
     const account = pathElement.getAccount()?.getAddress()
     const currency = pathElement.getCurrency()
+    if (account && currency) {
+      throw new XrpError(
+        XrpErrorType.MalformedProtobuf,
+        'PathElement protobuf should not contain both `account` and `currency` fields.',
+      )
+    }
+
     const xrpCurrency = currency && XrpCurrency.from(currency)
     const issuer = pathElement.getIssuer()?.getAddress()
+    if (account && issuer) {
+      throw new XrpError(
+        XrpErrorType.MalformedProtobuf,
+        'PathElement protobuf should not contain both `account` and `issuer` fields.',
+      )
+    }
+    // TODO check that `issuer` is omitted if the `currency` is XRP
```

Comment:
are you planning on doing this in the near future or is this still in progress? typically we try to include github name in TODOs, like `TODO(tedkalaw)`

Reply:
This requires some other issues with the gRPC implementation to be resolved. @amiecorso mentioned that there's likely an error with `PathElement` that currently makes this impossible to resolve atm.

Reply:
got it, sounds reasonable to me - can we add a task to track this?

Reply:
i'll add the task